packages/@okta/vuepress-site/docs/reference/rl-system-log-events/index.md

```diff
@@ -293,12 +295,119 @@ The following is an example System Log rate limit event where too many OTP verif
+                "state": null,
+                "country": null,
+                "postalCode": null,
+                "geolocation": {
+                    "lat": 36.62,
+                    "lon": -113.17
+                }
+            }
+        },
+        "authenticationContext": {
+            "authenticationProvider": null,
+            "credentialProvider": null,
+            "credentialType": null,
+            "issuer": null,
+            "interface": null,
+            "authenticationStep": 0,
+            "externalSessionId": "idx3GDqJrIfQhCRJTEuNC-l9A"
+        },
+        "displayMessage": "Operation rate limit violation",
+        "eventType": "system.operation.rate_limit.violation",
+        "outcome": {
+            "result": "DENY",
+            "reason": "Profile reload for john.smith@example.com skipped due to rate limiting"
+        },
+        "published": "2021-05-14T20:42:21.480Z",
+        "securityContext": {
+            "asNumber": null,
+            "asOrg": null,
+            "isp": null,
+            "domain": null,
+            "isProxy": null
+        },
+        "severity": "WARN",
+        "debugContext": {
+            "debugData": {
+                "operationRateLimitTimeUnit": "MINUTES",
+                "operationRateLimitScopeType": "user",
+                "deviceFingerprint": "3857b18395b101c0703feec226def544",
+                "operationRateLimitSecondsToReset": "96",
+                "requestId": "reqmXZkOgXpQdms4-3a8eD9mg",
+                "operationRateLimitThreshold": "1",
+                "operationRateLimitTimeSpan": "5",
+                "requestUri": "/idp/idx/identify",
+                "operationRateLimitType": "profile_reload",
+                "operationRateLimitSubtype": "AD agent",
+                "url": "/idp/idx/identify?"
+            }
+        },
+        "legacyEventType": null,
+        "transaction": {
+            "type": "WEB",
+            "id": "reqmXZkOgXpQdms4-3a8eD9mg",
+            "detail": {}
+        },
+        "uuid": "e19832a2-b4f4-11eb-9f1e-bba1874b8f01",
+        "version": "0",
+        "request": {
+            "ipChain": [
+                {
+                    "ip": "127.0.0.1",
+                    "geographicalContext": null,
+                    "version": "V4",
+                    "source": null
+                }
+            ]
+        },
+        "target": [
+            {
+                "id": "00u1ngpFSRLFie7vT0g4",
+                "type": "User",
+                "alternateId": "john.smith@example.com",
+                "displayName": "John Smith",
+                "detailEntry": null
+            }
+        ]
+    }
+```
+
 #### Operation rate limit subtypes
 
 The following table includes the available `Subtypes` for operation rate limits.
 
+> **Note:** The `AD agent` and `LDAP agent` subtypes are only available as a part of the Okta Identity Engine. Please [contact support](mailto:dev-inquiries@okta.com) for further information.
```

Comment:
**Note:** The `AD agent` and `LDAP agent` subtypes are only available with Okta Identity Engine.

Comment: Why would contacting Okta support help? What information could they provide? If they cannot provide any information, don't add the sentence asking users to contact support. Also, don't use "please".

Reply:
the text coming from https://oktawiki.atlassian.net/wiki/spaces/DOC/pages/2004649380/OIE+API+Reference+Documentation+Guidance, and I added the text based on the guidelines.

Removed the contact part and adopt your suggested text.